magenta-lib/src/main/scala/magenta/deployment_type/AmiCloudFormationParameter.scala

```diff
@@ -31,19 +37,51 @@ object AmiCloudFormationParameter
       val cloudFormationStackLookupStrategy =
         getCloudFormationStackLookupStrategy(pkg, target, reporter)
 
-      // TODO wrap this CloudFormation update in a ChangeSet. See `Cloudformation.scala`.
+      val changeSetName = s"${target.stack.name}-${new DateTime().getMillis}"
+
+      val stackLookup = new CloudFormationStackMetadata(
+        cloudFormationStackLookupStrategy,
+        changeSetName,
+
+        // We're updating an AMI. The stack should exist, so should never want to create it.
+        createStackIfAbsent = false
+      )
+
+      val stackTags = cloudFormationStackLookupStrategy match {
+        case LookupByTags(tags) => Some(tags)
+        case _                  => None
+      }
+
+      val amiLookupFn = getLatestAmi(pkg, target, reporter, resources.lookup)
+
+      val unresolvedParameters = new CloudFormationParameters(
+        target = target,
+        stackTags = stackTags,
+        amiParameterMap = amiParameterMap,
+        latestImage = amiLookupFn,
+
+        // Not expecting any user parameters in this deployment type
+        userParameters = Map.empty
```

Comment:
The `amiParametersToTags` parameter of the [`ami-cloudformation-parameter` deployment type](https://riffraff.gutools.co.uk/docs/magenta-lib/types#amicloudformationparameter) gets mapped to `amiParameterMap`. That is, we can assert that there are no other CFN parameters being adjusted.